setup.cfg

```diff
@@ -63,15 +63,14 @@ install_requires =
     psutil>=5.0
     psygnal>=0.3.4
     pydantic>=1.9.0
-    pygments>=2.4.0
+    pygments>=2.6.0
     PyOpenGL>=3.1.0
     PyYAML>=5.1
     qtpy>=1.10.0
-    scikit-image>=0.19.1
-    scikit-image[data]  # just `pooch`, but needed by `builtins` to provide all scikit-image.data samples 
+    scikit-image[data]>=0.19.1 # just `pooch`, but needed by `builtins` to provide all scikit-image.data samples
     scipy>=1.4.1 ; python_version < '3.9'
     scipy>=1.5.4 ; python_version >= '3.9'
-    sphinx<5  # numpydoc dependency. sphinx>=5 breaks the docs build; see https://github.com/napari/napari/pull/4915
+    sphinx>=4.3.0,<5  # numpydoc dependency. sphinx>=5 breaks the docs build; see https://github.com/napari/napari/pull/4915
```

Comment:
> BTW, not this PR necessarily, but do we really need this as a napari requirement with the split repo?
> I think we should make `[docs]` for the docs repo installation.

No, we could use a constraints file for that. In the next 2-3 days, I may have time to do that.

Reply:
This is not a docs requirement. Rather, we use numpydocs to move some docstrings around. Run `git grep numpydoc` to find where it's used. It might be something we can eventually deprecate.